### PR TITLE
fix(TypingTweaks): fix crash when using UK English

### PR DESCRIPTION
**Pull request by zt64**
Probably fixes other languages too, idk. I also made it use nicknames in guilds

### CODE REVIEW

File: src/plugins/typingTweaks.tsx [hunk 1]
```diff
@@ -105,7 +115,7 @@ export default definePlugin({
                     size={Avatar.Sizes.SIZE_16}
                     src={user.getAvatarURL(guildId, 128)}/>
             </div>}
-            {user.username}
+            {guildId ? GuildMemberStore.getNick(guildId, user.id) || user.username : user.username}
```
Comment (Vendicated) [inline src/plugins/typingTweaks.tsx:118]:
```suggestion
            {GuildMemberStore.getNick(guildId!, user.id) || user.username}
```
u can just

File: src/plugins/typingTweaks.tsx [hunk 2]
```diff
@@ -66,10 +66,18 @@ export default definePlugin({
         },
         // Changes indicator to format message with the typing users
         {
-            find: ',"SEVERAL_USERS_TYPING","',
+            find: "\"SEVERAL_USERS_TYPING\":\"",
```
Comment (Vendicated) [inline src/plugins/typingTweaks.tsx:69]:
```suggestion
            find: '"SEVERAL_USERS_TYPING":"',
```

File: src/plugins/typingTweaks.tsx [hunk 3]
```diff
@@ -66,10 +66,18 @@ export default definePlugin({
...
+                match: /("SEVERAL_USERS_TYPING"):".+?"/,
+                replace: "$1:\"**!!{a}!!**, **!!{b}!!**, and {c} others are typing...\""
+            },
+            predicate: () => settings.store.alternativeFormatting
+        },
+        {
+            find: ",\"SEVERAL_USERS_TYPING\",\"",
+            replacement: {
+                match: /(\i)\((\i),("SEVERAL_USERS_TYPING"),".+?"\)/,
+                replace: "$1($2,$3,\"**!!{a}!!**, **!!{b}!!**, and {c} others are typing...\")"
```
Comment (Vendicated) [inline src/plugins/typingTweaks.tsx:80]: do you even need this patch now

Comment (zt64) [reply src/plugins/typingTweaks.tsx:80]: yes, crashes on English US without it. do you love?

**Pull request merged**